client/network/src/config.rs

```diff
@@ -523,6 +523,9 @@ pub struct NetworkConfiguration {
 	/// a modification of the way the implementation works. Different nodes with different
 	/// configured values remain compatible with each other.
 	pub yamux_window_size: Option<u32>,
+
+	#[allow(missing_docs)]
+	pub qkd_addr: Vec<RpcAddrWithPeerId>,
```

Comment:
We cannot use this "as is", because this config stores `SocketAddr`, which contains only ip and port, while we need path as well, eq. `http://212.244.177.99:8282/api/v1/keys/Bob2SAE`. Second thing - it has "rpc" in it's name, and this is not rpc. Please, make this config more generic so we could use it for both configs or provide another config struct for QKD

Reply:
Done, could you please re-review this struct?